features/lorawan/lorastack/mac/LoRaMacCrypto.cpp

```diff
@@ -26,59 +26,33 @@
 #include <stdlib.h>
 #include <stdint.h>
 
-#include "mbedtls/aes.h"
-#include "mbedtls/cmac.h"
-
 #include "LoRaMacCrypto.h"
 #include "system/lorawan_data_structures.h"
 
 
 #if defined(MBEDTLS_CMAC_C) && defined(MBEDTLS_AES_C) && defined(MBEDTLS_CIPHER_C)
 
-/**
- * CMAC/AES Message Integrity Code (MIC) Block B0 size
- */
-#define LORAMAC_MIC_BLOCK_B0_SIZE                   16
-
-/**
- * MIC field computation initial data
- */
-static uint8_t mic_block_b0[] = {0x49, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-                                0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00};
-
-/**
- * Contains the computed MIC field.
- *
- * \remark Only the 4 first bytes are used
- */
-static uint8_t computed_mic[16];
-
-/**
- * Encryption aBlock and sBlock
- */
-static uint8_t a_block[] = {0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-                            0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00};
-
-static uint8_t s_block[] = {0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-                            0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00};
+LoRaMacCrypto::LoRaMacCrypto()
+{
+    memset(mic_block_b0, 0, sizeof(mic_block_b0));
```

Comment:
Could avoid explicit `memset`s by zero-initting arrays like any other element:

    LoRaMacCrypto::LoRaMacCrypto() : mic_block_b0(), a_block(), ...

You will still need to manually set the non-zero elements though (with current C++03).

Reply:
Ok. I'll change this.